README.md

```diff
@@ -1,9 +1,12 @@
 avr-device [![crates.io page](https://img.shields.io/crates/v/avr-device.svg)](https://crates.io/crates/avr-device) [![docs.rs](https://docs.rs/avr-device/badge.svg)](https://docs.rs/avr-device) ![Continuous Integration](https://github.com/Rahix/avr-device/workflows/Continuous%20Integration/badge.svg)
 ==========
+
```

Comment:
I'd prefer if you drop the newlines that were added to README.md from your diff.  It doesn't really make sense to do such changes in a commit related to ATmega16...

(Also I personally prefer keeping markdown text more compact, without newlines after each heading, but that's a perference thing)